### ISSUE TITLE
Unexplained crash after extended use

### ISSUE THREAD

**Issue by Sibusten** [Sibusten/derpibooru-downloader#2]
Program has crashed on me several times after extended use (anywhere from one day to several) and running many downloads over that time.
Unsure what is causing it, could be a memory leak or some other unhandled exception. Process does seem to grow in memory after each run.

Will have to look more into it sometime.

**Comment by fionnafire** [comment]
When I start the EXE I get this error: "This application failed to start because it could not find or load the Qt platform plugin "windows""

![derpidown1](https://user-images.githubusercontent.com/30191419/28236763-6389aed4-68e3-11e7-9e9c-29b7ed8fbc80.PNG)
![derpidown2](https://user-images.githubusercontent.com/30191419/28236765-66586fa6-68e3-11e7-9571-589140bfbaab.PNG)

I did some googling but even after adding "qwindows.dll"  in a subfolder labeled "platforms" it still won't get past the above error.

![derpidown3](https://user-images.githubusercontent.com/30191419/28236766-67a904c4-68e3-11e7-87a4-4e6a73d727a9.PNG)

**Comment by Sibusten** [comment]
Thank you for bringing this to my attention, I seem to have forgotten to include that in any of the releases I did. I've added it to the latest release.

Not all dlls are the same, even if they have the same name, so please try it with the one I provided and let me know if it works.

[https://github.com/Sibusten/derpibooru-downloader/releases/tag/v1.3.1](https://github.com/Sibusten/derpibooru-downloader/releases/tag/v1.3.1)

**Comment by fionnafire** [comment]
yes! it works perfectly now! I've been waiting for a program like this to be made for like two years! thank you!!

**Comment by Sibusten** [comment]
You're welcome, I am glad it is working properly now.

If you have any further comments or questions, feel free to open up an issue for them. Thank you again.

**Comment by EpicLPer** [comment]
I think you should close this issue so others won't think there still is one ;)

**Comment by Sibusten** [comment]
There is still an issue, with the original crash related to memory leaking. I don't keep the program open long enough to know if it's still there, but I never fixed it. The rest of the posts are about an unrelated crash that should have been it's own issue, but I just answered it here instead of telling them to move it.

I changed the title to reflect this.

**Comment by fionnafire** [comment]
Sorry. I didn't really know how GitHub worked.

**Comment by Sibusten** [comment]
It's alright, no big deal. For future reference, there is a green "New Issue" button in the top right that you can use to start a new issue.
